src/pages/workspace/accounting/netsuite/advanced/NetSuiteAdvancedPage.tsx

```diff
@@ -38,7 +44,7 @@ type MenuItemWithSubscribedSettings = Pick<MenuItem, 'type' | 'description' | 't
 function NetSuiteAdvancedPage({policy}: WithPolicyConnectionsProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const policyID = policy?.id ?? '-1';
+    const policyID = policy?.id ?? `${CONST.DEFAULT_NUMBER_ID}`;
```

Comment:
This still violates the link rule. Please check https://expensify.slack.com/archives/C01GTK53T8Q/p1737913879440099

Reply:
Thanks for pointing that out @s77rt 🙏